Update input provider readme (#2042)
BBC-archive/psammead: packages/utilities/psammead-storybook-helpers/README.md

`LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.

> `inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 props, `slots` and `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.

Reviewer: ```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

> `inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`.Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.

Author: ```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```

> `inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.

Reviewer 2: >Takes 4 arguments

Really it takes 1 arguement, an object with 4 possible keys 🙈

Reviewer: Good spot @dr3

> `inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 4 arguments, `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.

Author: ```suggestion
`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes 1 argument of an object, with these possible keys: `slots`, `componentFunction`, `services` and `options`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet). Returns the return value of `componentFunction`. This should usually be a React Component.
```
